app/views/index.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Welcome to Chitter!</h1>
+
+<form action="/sign_up" method='get'>
+  <input type="submit" name="" value="Sign-up">
+</form>
+
+<form action="/log_in" method='get'>
+  <input type="submit" name="" value="Log-in">
+</form>
```

Comment:
I'm interested that you're using `get` requests here, when you're creating a new kind of resource on both these forms ('creating' is a RESTful `post` method).